android/gradle.properties

```diff
@@ -17,8 +17,3 @@ KlaviyoReactNativeSdk_minSdkVersion=23
 KlaviyoReactNativeSdk_targetSdkVersion=31
 KlaviyoReactNativeSdk_compileSdkVersion=31
 KlaviyoReactNativeSdk_ndkversion=21.4.7075529
-
-# For local SDK development: set to true in your local.properties file
-# you must set separately for the example app and the library
-localCompositeBuild=false
-localSdkPath=../../klaviyo-android-sdk
```

Comment:
Dan astutely pointed out that there's no reason to declare these here, they only need to live in local.properties.